### PR TITLE
feat: add authentication restriction

### PR DESCRIPTION
**Pull request by thisham**
## Testings
- [x] Create doctor data
- [x] Read all doctor data

### CODE REVIEW

**Review by dhichii — APPROVED** [review]
Nice Job!

File: src/routes/route.go [hunk 1]
```diff
@@ -18,61 +18,57 @@ func New() *echo.Echo {
...
 	admin.DELETE("/:id", caHandler.Admin.RemoveAdminByIDHandler)
 
-	// queue
-	queue := route.Group("/queues")
-	queue.POST("", caHandler.Queue.CreateQueueHandler)
+	queue := authenticatedRoute.Group("/queues")
+	queue.POST("", caHandler.Queue.CreateQueueHandler, middlewares.GrantAdmin)
 	queue.GET("", caHandler.Queue.ShowAllQueuesHandler)
 	queue.PUT("/:id", caHandler.Queue.AmendQueueByIDHandler)
 	queue.DELETE("/:id", caHandler.Queue.RemoveQueueByIDHandler)
```
Comment (dhichii) [inline src/routes/route.go:66]: Prefer only the admin can delete the queue

**Review by dhichii — CHANGES_REQUESTED** [review]

File: src/routes/route.go (hunk 1 above)
Comment (thisham) [reply src/routes/route.go]: OK, https://github.com/ALTERA-CAPSTON-41/backend/pull/55/commits/f57ab04b6e6477fb40bc0650fad3c14a6ab3a1cc.

**Review by dhichii — APPROVED** [review]
Let's merge ;)

**Pull request merged**